egs2/mucs21_subtask1/asr1/local/data.sh

```diff
@@ -0,0 +1,9 @@
+
```

Comment:
It's OK to directly run the script. But maybe better with logs and interaction with `db.sh`. Could you please modify this? You may refer to this for an example: https://github.com/espnet/espnet/blob/master/egs2/commonvoice/asr1/local/data.sh